### ISSUE TITLE
When using detailslist with groups , div with role="row" is inside the div with role="listitem" . 

### ISSUE THREAD

**Issue by mee-1993** [OfficeDev/office-ui-fabric-react#1723]
### Bug Report

- __Package version(s)__: (2.20.3)
- __Browser and OS versions__: (fill this out if relevant)

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (No)

Requested priority: (Normal) 

Products/sites affected: (if applicable)

#### Describe the issue:
Div with role="row" is enclosed within div with role="listitem" . 

#### Expected behavior:
http://www.w3.org/TR/wai-aria/roles#row  should be followed .
